File: web-film-backend/src/main/java/com/tamdao/web_film_backend/dto/response/AIChatResponse.java
package com.tamdao.web_film_backend.dto.response;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;
import org.springframework.data.domain.Page;

@Data
@Builder
@NoArgsConstructor
@AllArgsConstructor
public class AIChatResponse {
    private boolean isMovieQuery;
    private String aiMessage;
    private Page<MovieResponse> movies;
}

File: web-film-android/app/src/main/java/com/tamdao/cinestream/feature/ai_chat/model/AIChatResponse.kt
package com.tamdao.cinestream.feature.ai_chat.model

import com.tamdao.cinestream.data.model.MovieDto
import com.tamdao.cinestream.data.model.PageResult

/**
 * Response payload from the AI Chat API.
 */
data class AIChatResponse(
    val isMovieQuery: Boolean,
    val aiMessage: String,
    val movies: PageResult<MovieDto>?
)
